doc/crm.8.adoc

```diff
@@ -2104,6 +2104,71 @@ Example:
 utilization xen1 set memory 4096
 ...............
 
+[[cmdhelp.sbd,SBD management]]
+=== `sbd` - SBD management
+
+This level is for managing the SBD (STONITH Block Device) daemon.
+
+[[cmdhelp.sbd.configure,configure SBD]]
+==== `configure`
+
+Configure the SBD daemon for both disk-based and disk-less mode.
+
+Main functionailities include:
+- Show configured disk metadata
+- Show contents of /etc/sysconfig/sbd
+- Show SBD related cluster properties
+- Newly setup SBD configuration on a running cluster
+- Update the existing parameters
+- Add more devices to the existing disk-based SBD configuration
+
+For more details on SBD and related parameters, please see man sbd(8).
+
+Usage:
+...............
+# For disk-based SBD
+crm sbd configure show [disk_metadata|sysconfig|property]
+crm sbd configure [device=<dev>]... [watchdog-device=<dev>] [watchdog-timeout=<integer>] [allocate-timeout=<integer>] [loop-timeout=<integer>] [msgwait-timeout=<integer>]
+
+# For disk-less SBD
+crm sbd configure show [sysconfig|property]
+crm sbd configure device="" [watchdog-device=<dev>] [watchdog-timeout=<integer>]
```

Comment:
When converting the disk-based SBD to the diskless SBD, fence_sbd primitive should be removed too. Currently not yet.